opentracing/src/main/java/opentracing/TraceCodec.java

```diff
@@ -24,16 +24,16 @@
  * binary message format, so that it can be inspected efficiently by the middleware / routing layers
  * without parsing the whole message.
  */
-public interface TraceContextCodec {
+public interface TraceCodec {
```

Comment:
I'm unsure about the need for this class.

What is the need to expose serialisation details to the implementer?